File: 01_gpio_digital/led_blink.py
import RPi.GPIO as GPIO
import time

ledPin = 4
GPIO.setmode(GPIO.BCM) # GPIO.BCM or GPIO.BOARD
GPIO.setup(ledPin, GPIO.OUT) # GPIO.OUT or GIPo.in

for i in range(10):
    GPIO.output(ledPin, GPIO.HIGH) # 1
    print("LED ON")
    time.sleep(1)
    GPIO.output(ledPin, GPIO.LOW) # 0
    print("LED OFF")
    time.sleep(1)

GPIO.cleanup()

File: 01_gpio_digital/led_blink.c
#include <stdio.h>
#include <wiringPi.h>
#define LED_PIN 4

int main (void){
    //wiringPisetup();
    wiringPiSetupGpio();
    pinMode (LED_PIN, OUTPUT);

    for (int i = 0; i < 5; i++){
        printf("LED ON\n");
        digitalWrite (LED_PIN, HIGH) ; delay(1000);
        
        printf("LED OFF\n");
        digitalWrite (LED_PIN, LOW) ; delay(1000);  
    }

    return 0;
}